ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/MLChatAgentRunner.java

```diff
@@ -417,7 +431,9 @@ private void runReAct(
                                                 ModelTensor
                                                     .builder()
                                                     .name("response")
-                                                    .dataAsMap(ImmutableMap.of("response", finalAnswer))
+                                                    .dataAsMap(
+                                                        ImmutableMap.of("response", finalAnswer, ADDITIONAL_INFO_FIELD, additionalInfo)
```

Comment:
In case of flow agent and React agent additional info is added to ModelTensors list instead of adding to `dataAsMap` : https://github.com/opensearch-project/ml-commons/blob/feature/agent_framework_dev/ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/MLReActAgentRunner.java#L537

We will need to follow same output schema for all agent types.

Reply:
@arjunkumargiri to be consistent with `getInteraction` API https://github.com/opensearch-project/ml-commons/blob/feature/agent_framework_dev/plugin/src/main/java/org/opensearch/ml/rest/RestMemoryGetConversationsAction.java#L34 response data structure, it's better to add to `dataAsMap`